contrib/cirrus/cache_release.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+source $(dirname $0)/lib.sh
+
+req_env_var GOSRC CIRRUS_BUILD_ID
+
+cd $GOSRC
+
+make release_info.txt
+FIELDS="$RELEASE_BASENAME $RELEASE_VERSION $RELEASE_DIST $RELEASE_DIST_VER $RELEASE_ARCH $RELEASE_FILE"
```

Comment:
You probably need to remove the dollar signs before this could work.

Reply:
Woops, good point, :laughing:

Reply:
got it